meetings/2025-05-13-taskforce-meeting.md

```diff
@@ -0,0 +1,87 @@
+- (discussion of: who created the label is different from who applied a label somewhere)
+- Darius: so we have talked about Objects a lot but not Activity, so maybe the Activity tells us who is applying a label to an object, but the Object itself is just containing information about who created the label originally
+- Emelia: so maybe we focus on Object scope for a current draft and we leave the Activity scope for the future. I don't think it would be useful for someone to say "I put a label on my own post" to be discovered through a side channel.
+- Darius: we have 9 minutes left. Do we have action items we want to do?
+- Emelia: the next step is to say there are two cases here. One is the content warning or label applied at authorship time, and the other is a content warning or label or other text applied after the fact.
+- Darius: essentially first party vs third party labeling.
+- Darius: is there a server Actor specified somewhere? Are we going to point to a big blank space and say "please specify this everyone"?
+- a: there needs to be at least a way to say "this Actor is managed by this Service"
+- Emelia: yes we need a way to say "which moderators are behind this Actor". we have an instance Actor that is a Service type, but hidden from users.
+- a: there can be multiple such actors for different purposes. Most common is a proxy for signing http messages, there are relays, etc.
+- Emelia: I think we should write a FEP.
+- Jaz: in something Darius just said, we probably dont' want individual human moderators identifying. More like teams, institutions, etc.
+- Jaz: for me a desired outcome is: `summary` returns to being a summary, and it can be a summary, or CW, or leadin to a joke, or whatever. It happens at authorship time and can and does serve that purpose now. A world in which people have authorship, a summary field they are free to use or not and software that interprets it how it wants, and we keep that separate from structured labels. We already tag our content with hashtags. I can see a world in which software clients give me the option to let me use the vocabulary of another server to help me tag my content. The larger point: I would love it if `summary` remained a summary and people can write whatever they want and that definitionally is not a content warning. Mastodon does the Hazard Tape and CW thing, but if we do this that could stop happen.
+- Emelia: do we need an actual text property for content warning, or do we want the summary to BE the content warning?
+- Jaz: why can't I personally write a `summary` of "Trigger warning: content includes blah" -- I use Trigger warning specifically because that's different from CW.
+- Emelia: if we say `summary` is that, and then labels actually drive content warnings, how do you have software indicate "the author meant this as a CW" vs "the author meant to collapse".
```

Comment:
```suggestion
- Emelia: Do we need an actual text property for content warning, or do we want the summary to BE the content warning?
- Jaz: Why can't I personally write a `summary` of "Trigger warning: content includes blah" I use Trigger warning specifically because that's different from CW.
- Emelia: If we say `summary` is that, and then labels actually drive content warnings, how do you have software indicate "the author meant this as a CW" vs "the author meant to collapse".
```